documentation/src/main/resources/pages/ditto/basic-metadata.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Metadata Support
+keywords: metadata, things
+tags: [model]
+permalink: basic-metadata.html
+---
+
+Besides [features](basic-feature.html) a Thing in Ditto is also able to store Metadata information. 
+These informations can be additional information for describing specific features or properties of the thing.
+A very simple example is the timestamp when the current value of the feature or property was set.
+
+Metadata can be manipulated by using the `metadata` header, see [here](protocol-specification.html#headers) for more information.
+
+## Predefined Metadata 
+
+Currently, there are no pre-defined Metadata keys.
+
+<!--
+Right now, the only fixed pre-defined Metadata key is `issuedAt`.
+
+### The `issuedAt` Metadata key
+ 
+The Metadata key `issuedAt` describes the timestamp when the current value of a feature property was _recorded_ (_not when it was set_).
+This means, that a client _can_ send this value by using the `metadata` header. If the client does not send
+this header the `issuedAt` field will be set by Ditto with the timestamp when the value was written.
+-->
+
+## Setting custom Metadata
+
+The only way to set arbitrary `Metadata` is by using the `put-metadata` header, see [here](protocol-specification.html#headers) for more information.
+The Format of the Json Array is 
+
+```
+[
+  {
+    "key":"/features/lamp/properties/color/r",
+    "value":{"issuedAt":someTimestamp,"issuedBy":{"name":"me","mail":"me@mail.com"}}
```

Comment:
Is that really possible? According to https://github.com/eclipse/ditto/issues/785#issuecomment-688062943 I'd assume that each metadata should receive it's own array entry?

Reply:
Hey, I'm refering to https://github.com/eclipse/ditto/issues/680#issuecomment-675446480 here. I think I tested it but have to retest it to be sure.

Reply:
should be, yes - you set an JsonObject for the `"/features/lamp/properties/color/r"` property.